src/components/activity-list/ActivityList.js

```diff
@@ -97,17 +102,15 @@ const ActivityList = ({
     }
     return currentPendingTransactionsCount;
   }, [sections, requests]);
+
+  const { colors } = useTheme();
+  const renderSectionHeaderWithTheme = useCallback(
+    ({ section }) => renderSectionHeader({ colors, section }),
+    [colors]
+  );
+
   return network === networkTypes.mainnet || sections.length ? (
-    recyclerListView ? (
```

Comment:
can you help me understand this?

Reply:
We discussed long ago that this is now effectively dead code and can be removed.